packages/hosted-form-v2/src/iframe-content/hosted-input-validator.ts

```diff
@@ -127,7 +127,14 @@ export default class HostedInputValidator {
     }
 
     private _getCardNameSchema(): StringSchema {
-        return string().max(200).required('Full name is required');
+        return string()
+            .max(200)
+            .required('Full name is required')
+            .test({
+                message: 'Credit card name must be valid',
```

Comment:
Is this customer facing message? If yes, then how do we translate this?

Reply:
The error name `invalid_card_name` is mapped to the translated error message in checkout-js like [here](https://github.com/bigcommerce/checkout-js/blob/cf153bbd08b86bcffb7beea5ba41de6594b62ab7/packages/core/src/app/payment/hostedCreditCard/getHostedCreditCardValidationSchema.ts#L54).